### PR TITLE
Added data for EO 1-100

### PR DESCRIPTION
**Pull request by Claided**
Edited and Added Mob data for EO 1-50. Added Mob data and EN localization for EO 51-100. Updated EO 1-100 Floor data with info I find useful. Mob Data based on personal experience, Maygi's guide, and Meifa's EO QuickRef spreadsheet. 
First time doing a pull request, so I hope it helps!

### CODE REVIEW

**Comment by wolfcomp** [comment]
Please redo your pr to use the newest data commits as these have conflicts

**Comment by Claided** [comment]
Sorry about that, I didn't see your latest commits. I think the conflicts have been resolved and it should be able to be merged.

File: EO/51-100.yml
Comment (wolfcomp) [inline EO/51-100.yml:1]: Would gladly take that you redo this file completely to what is done in the main data branch

File: Localization/en/EO/51-100.yml
Comment (wolfcomp) [inline Localization/en/EO/51-100.yml:1]: This file is not properly formatted and needs to be looked at since it still contains merge conflict parts

**Review by wolfcomp — CHANGES_REQUESTED** [review]
Still not fully done with what is done with the file for what is on the main data branch

**Pull request closed without merging**